src/classes/Bot.ts

```diff
@@ -61,7 +60,7 @@ export default class Bot {
 
     readonly tf2gc: TF2GC;
 
-    readonly handler: Handler;
+    readonly handler: MyHandler;
```

Comment:
Hurm you sure? 🤔

Reply:
We create new MyHandler() in the constructor so it should be safe.

Reply:
See https://github.com/TF2Autobot/tf2autobot/pull/252/files#diff-450b22a2f8e3cb5c78358968ec6eed9abf6f1c6e6719786a39f467777e7e5d83R162

the handler is a MyHandler type, might as well use it as that

Reply:
alrighty